README.md

```diff
@@ -124,6 +124,27 @@ ssl_certificate_key /path/to/certificate-key.pem;
 
 Also, `proto=https` option is needed for the `listen` directive.
 
+#### Self-signed certificate genration
+
+In case of using self-signed certificate with Tempesta, it's usefull
+to use openSSL to generate the key and certificate. Use the following
+shell command:
```

Comment:
In case of using a self-signed certificate with Tempesta, it's convenient to use OpenSSL to generate the key and the certificate. The following shell command can be used: